chpl/chpl-api/e2e/data/chplIds-test-data.json

```diff
@@ -1,34 +1,17 @@
 [
- {
-  "certifiedProductId":4770,
-  "chplid":"CHP-028037"
- },
- {
-  "certifiedProductId":4609,
-  "chplid":"CHP-028396"
- },
- {
-  "certifiedProductId":3321,
-  "chplid":"CHP-009321"
- },
- {
-  "certifiedProductId":3853,
-  "chplid":"CHP-006605"
- },
- {
-  "certifiedProductId":9437,
-  "chplid":"15.04.04.2214.Proc.14.00.0.180427"
- },
- {
-  "certifiedProductId":10274,
-  "chplid":"15.05.05.1292.CORT.01.00.1.200114"
- },
- {
-  "certifiedProductId":10509,
-  "chplid":"15.04.04.3082.Mine.0V.00.0.201229"
- },
- {
-  "certifiedProductId":8883,
-  "chplid":"15.02.02.2391.A046.01.00.1.171109"
- }
+	{
+		"certifiedProductId": 3321,
+		"certifiedlistings2015": "15.04.04.2214.Proc.14.00.0.180427",
+		"certifiedlistings2014": "CHP-028396"
+	},
+	{
+		"certifiedProductId": 4770,
+		"certifiedlistings2015": "15.04.04.2214.Proc.14.00.0.180427",
```

Comment:
15.04.04.2214.Proc.14.00.0.180427- This listing is used twice in the file

Reply:
Duplicate id is removed and added another 2015 listing id